draft-ietf-core-senml.md

```diff
@@ -346,10 +346,15 @@ If either the Base Time or Time value is missing, the missing attribute is
 considered to have a value of zero. The Base Time and Time values are added
 together to get the time of measurement. A time of zero indicates that the
 sensor does not know the absolute time and the measurement was made roughly
-"now". A negative value is used to indicate seconds in the past from roughly
+"now". Zero, or a negative value is used to indicate seconds in the past from roughly
```

Comment:
While the two definitions of zero (this and previous sentence) result in same time ("now"), I found adding the zero here bit confusing, so I would not recommend this change.